vtex/my-subscriptions
# Action bar - order and payment [last subscription cycle]

#### What does this PR do? \*
New action bar behaviors for the last subscription's cycle.

#### How to test it? \*
- Login in [workspace](https://bambokian--recorrenciacharlie.myvtex.com/account#/subscriptions)
- Visit a specific subscription details screen
- Then, check 'contextual primary actions' section on Figma [layout](https://www.figma.com/file/cjH10m6QuEYf8YU08ydhOFUF/Subscriptions-Storefront?node-id=2585%3A163) and compare with the orders and payments action bar present in the workspace

#### Related to / Depends on \*
